lib/static_sync/storage.rb

```diff
@@ -49,6 +49,7 @@ def storage
         :region                => @config.remote['region'],
         :aws_access_key_id     => @config.remote['username'],
         :aws_secret_access_key => @config.remote['password'],
+        :aws_session_token     => @config.remote['session_token'],
```

Comment:
If the `session_token` doesn't exist we'll pass in a `nil` value here. I just had a look at the `fog-aws` gem and this is how the session token is set:

```ruby
if @aws_session_token
  params[:query]['X-Amz-Security-Token'] = @aws_session_token
end
```

If `@aws_session_token` is nil, this condition will be false. I couldn't find any code that checks for any keys with nil values so both username and password attributes are being set independently from the session token.